client/js/packages/packageApi.js

```diff
@@ -0,0 +1,24 @@
+// Our public API
+window.TheLounge = {
+};
```

Comment:
I think I should probably add at least 1 tiny thing in here so I can test this stuff works, something like ... maybe ... the version of the lounge that we are running against maybe?

Reply:
>maybe ... the version of the lounge that we are running against maybe

Would be useful, IMO